### PR TITLE
Login Logout

### PR DESCRIPTION
**Pull request by BichVN**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else  
+      flash[:danger] = t("login_failed_mess")
+      render 'new'
```
Comment (phuongbk) [inline app/controllers/sessions_controller.rb]: Change ' to "

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -5,6 +5,24 @@
       <ul class="nav navbar-nav navbar-right">
         <li><%= link_to t("home_link"), root_path %></li>
         <li><%= link_to t("help_link"), help_path %></li>
+        <% if logged_in? %>
+          <li><%= link_to t("users_link"), '#' %></li>
```
Comment (phuongbk) [inline app/views/layouts/_header.html.erb]: change ' to "

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -5,6 +5,24 @@
...
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
```
Comment (phuongbk) [inline app/views/layouts/_header.html.erb]: Use link_to

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -5,6 +5,24 @@
...
+             <%= t("account") %><b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li><%= link_to t("profile"), current_user %></li>
+              <li><%= link_to t("setting"), '#' %></li>
```
Comment (phuongbk) [inline app/views/layouts/_header.html.erb]: ' -> "

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<% provide(:title, t("login_title") ) %>
+<h1><%= t("login_title") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
+      <%= f.label :password %>
+      <%= f.password_field :password, class: 'form-control' %>
```
Comment (phuongbk) [inline app/views/sessions/new.html.erb]: ' -> "

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+      <%= f.submit t("login_title"), class: "btn btn-primary" %>
+    <% end %>
+    <p>New user? <%= link_to t("signup_button"), signup_path %></p>
```
Comment (phuongbk) [inline app/views/sessions/new.html.erb]: I18n

File: app/views/users/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <aside class="col-md-4">
+    <section>
+      <h1>Username: <%= @user.name %></h1>
+      <h1>Email: <%= @user.email %></h1>
+    </section>
+  </aside>
+</div>
```
Comment (phuongbk) [inline app/views/users/show.html.erb]: I18n

File: app/controllers/users_controller.rb [hunk 8]
```diff
@@ -6,13 +6,18 @@ def new
   def create
     @user = User.create user_params
     if @user.save
-      flash[:sucsess] = t("signin_success_mess")
+      log_in @user
+      flash[:sucsess] = t("signup_success_mess")
```
Comment (manhnhhedspi) [inline app/controllers/users_controller.rb]: remove ()

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -5,6 +5,25 @@
...
+              <li><%= link_to t("profile"), current_user %></li>
+              <li><%= link_to t("setting"), root_path %></li>
+              <li class="divider"></li>
+              <li>
+                <%= link_to t("logout"), logout_path, method: "delete" %>
+              </li>
+            </ul>
+          </li>
+        <% else %>
+          <li><%= link_to "Log in", login_path %></li>
```
Comment (manhnhhedspi) [inline app/views/layouts/_header.html.erb]: I18N

File: app/views/sessions/new.html.erb [hunk 10]
```diff
@@ -0,0 +1,18 @@
+<% provide(:title, t("login_title")) %>
```
Comment (manhnhhedspi) [inline app/views/sessions/new.html.erb]: remove ()

File: app/views/users/new.html.erb [hunk 11]
```diff
@@ -1,5 +1,5 @@
 <% provide(:title, t("signup_title")) %>
```
Comment (manhnhhedspi) [inline app/views/users/new.html.erb]: remove ()

**Pull request merged**